### PR TITLE
Create pt.json

### PR DESCRIPTION
**Pull request by guschaves**
I did translate the texts to portuguese, if you implement that in the app it would be very good for us, brazilians.

### CODE REVIEW

File: lib/translations/pt.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        "fontSize": "Tamanho da fonte",
+        "language": "Idioma"
+    },
+    "languages": {
+        "en": "English",
+        "fr": "French",
+        "zh": "Chinese",
+	"es": "Spanish",
+	"pl": "Polaco"
+	"pt": "Português"
```
Comment (shprink) [inline lib/translations/pt.json]: Tiny problem here you forgot a colon.

**Comment by guschaves** [comment]
Can you change that if you add on the app development? Because I don't know how to fix it

**Comment by shprink** [comment]
Thanks @guschaves :)

**Comment by guschaves** [comment]
Thank you, count on me if you need any translation for pt-br.

**Pull request merged**